src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -641,6 +641,17 @@ public boolean verifyNewRankRecipientsQuestionFormIsDisplayed() {
         return contribForm.isDisplayed() && addNewQuestionButton.isDisplayed();
     }
 
+    /*
+     * checks if alert class is enabled on the visibility options div for the specified question number.
+     * To be used only with questions with responses.
+     */
+    public boolean verifyAlertClassIsEnabledForVisibilityOptions(int questionNo) {
+        final String visibilityOptionsDivXPath = "//div[@id='questionTable-" + questionNo + "']//div[@class='panel-body']"
+                + "//div[contains(@class, 'col-sm-12 margin-bottom-15px padding-15px')]";
```

Comment:
`col-sm-12 margin-bottom-15px padding-15px` isn't a very meaningful identifier. Can instead search for the element with the `visibility-title` class and take its parent's parent?